targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F413xH/TARGET_MTS_DRAGONFLY_F413RH/ONBOARD_TELIT_HE910.cpp

```diff
@@ -17,7 +17,7 @@
 #if MBED_CONF_NSAPI_PRESENT
 
 #include "gpio_api.h"
-#include "platform/mbed_wait_api.h"
+//#include "platform/mbed_wait_api.h"
```

Comment:
please just remove this line

Reply:
I have cleaned up these issues and amended the commit.